app/Http/Controllers/Api/SentinelController.php

```
@@ -0,0 +1,146 @@
+                'team_id' => $server->team_id,
+            ]);
+
+            return response()->json(['message' => 'Unauthorized'], 401);
+        }
+        $data = $request->all();
+
+        // Heartbeat MUST update on every push — drives isSentinelLive() and SSH-check skipping.
+        $server->sentinelHeartbeat();
+
+        if ($this->shouldDispatchUpdate($server, $data)) {
+            PushServerUpdateJob::dispatch($server, $data);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\App\Jobs\PushServerUpdateJob' in method 'push'. (undefined)

(StaticAccess)